src/sentry/replays/query.py

```diff
@@ -184,7 +187,60 @@ def query_replays_count(
     )
 
 
-# Select.
+def query_replays_dataset_tagkey_values(
+    project_ids: List[str],
+    start: datetime,
+    end: datetime,
+    environment: str | None,
+    tag_key: str,
+):
+    """Query replay tagkey values. Like our other tag functionality, aggregates do not work here."""
+
+    query_options = {"limit": Limit(1000)}
+    where = []
+
+    if environment:
+        where.append(Condition(Column("environment"), Op.IN, environment))
+
+    value = TAG_QUERY_ALIAS_COLUMN_MAP.get(tag_key)
+    if value is None:
+        group_by_param = Function(
+            "arrayJoin",
+            parameters=[all_values_for_tag_key(tag_key, Column("tags.key"), Column("tags.value"))],
+            alias="tag_value",
+        )
+        value = Column("tag_value")
```

Comment:
Likewise this could be named better.  Since its not being used elsewhere I think `group_by_param` or `grouped_column` may be appropriate.